prisma/schema.prisma

```diff
@@ -1,39 +1,41 @@
-// This is your Prisma schema file,
-// learn more about it in the docs: https://pris.ly/d/prisma-schema
-
-// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
-// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
-
 generator client {
   provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider = "mongodb"
+  url      = env("DATABASE_URL") // Use your MongoDB connection string
 }
 
 model User {
-  id    String  @id @default(dbgenerated()) @map("_id")
+  id    String  @id @default(auto()) @map("_id") @db.ObjectId
   email String  @unique
   name  String?
   posts Post[]
 }
 
 model Post {
```

Comment:
What are posts?
Are we supporting this in any of our releases or is this optional?